Lib/tkinter/__init__.py

```diff
@@ -3745,27 +3745,34 @@ def compare(self, index1, op, index2):
         return self.tk.getboolean(self.tk.call(
             self._w, 'compare', index1, op, index2))
 
-    def count(self, index1, index2, *options): # new in Tk 8.5
+    def count(self, index1, index2, *options, return_ints=False): # new in Tk 8.5
```

Comment:
While Tk versions older than 8.5 and even 8.6 are no longer supported, it may be useful from historical perspective. Some old code written before Tk 8.5 can give benefit be using this method, and the explanation why it is not used maybe because it is too old.